tests/test_ingest_external_file.rs

```diff
@@ -432,3 +432,34 @@ fn test_mem_sst_file_writer() {
     assert!(env.delete_file(mem_sst_str).is_ok());
     assert!(env.file_exists(mem_sst_str).is_err());
 }
+
+#[test]
+fn test_set_external_sst_file_global_seq_no() {
+    let db_path = TempDir::new("_rust_rocksdb_set_external_sst_file_global_seq_no_db").expect("");
+    let db = create_default_database(&db_path);
+    let path = TempDir::new("_rust_rocksdb_set_external_sst_file_global_seq_no").expect("");
+    let file = path.path().join("sst_file");
+    let sstfile_str = file.to_str().unwrap();
+    gen_sst(
+        ColumnFamilyOptions::new(),
+        Some(db.cf_handle("default").unwrap()),
+        sstfile_str,
+        &[(b"k1", b"v1"), (b"k2", b"v2")],
+    );
+
+    let handle = db.cf_handle("default").unwrap();
+    let seq_no = 1;
+    // varify change seq_no
+    let r1 = set_external_sst_file_global_seq_no(&db, &handle, sstfile_str, seq_no);
+    assert!(r1.is_ok() && r1.unwrap() != seq_no);
+    // varify that seq_no are equal
+    let r2 = set_external_sst_file_global_seq_no(&db, &handle, sstfile_str, seq_no);
+    assert!(r2.is_ok() && r2.unwrap() == seq_no);
+
+    // change seq_no back to 0 so that it can be ingested
```

Comment:
If global_seq_no is not equal 0, is it can be ingested?

Reply:
No, it can't, the rocksdb will do a test to check whether the `seq_no` is corrupted when it is doing ingestion job.

Reply:
Seems it should work even if seqno is not 0, what's the error you encounter?

Reply:
`external file have non zero sequence number`